src/unittest/python/test_api.py

```diff
@@ -20,42 +21,39 @@ def server_is_up():
         else:
             # Quasi check for server status
             return all([bool(txt) for txt in ['api', 'is', 'online'] if txt in str(status_request)])
```

Comment:
this is something for next round of fixes, but we shouldn't be calling out to the API in any of our unit tests.  I'll work on that when making things local